icicle-compiler/src/Icicle/Sea/IO/Psv/Output.hs

```diff
@@ -411,6 +415,10 @@ seaOfOutput isJSON struct structIndex outId missing env outType argTypes transfo
    seaOfOutputBase' b
      = seaOfOutputBase b mismatch
 
+seaOfNotAnError :: Doc -> Doc
+seaOfNotAnError v =
+  "((" <> v <> " == ierror_not_an_error) || (" <> v <> " == ierror_tombstone))"
```

Comment:
hm, so I made this change because I wanted this behaviour:

input:
```
homer|test|tombstone|2000-1-1
```

core:
```
test_query
STREAM test
...
KONS = False
```

output:
```
homer|test_query|False
```